mobly/controllers/android_device.py

```diff
@@ -408,6 +408,8 @@ def start_services(self, skip_sl4a=False):
         except:
             self.log.exception("Failed to start adb logcat!")
             raise
+        if skip_sl4a is None:
+            skip_sl4a=getattr(ad, KEY_SKIP_SL4A, False)
```

Comment:
What's the benefit in removing it?
Also, start_services could be called multiple times in the lifetime of ad.